tests/unit/worker/task_router_test.py

```diff
@@ -107,3 +108,14 @@ def test_default_exception_handler_logs_a_warning(mocked_job_with_adapter: Job):
 
         mocked_job_with_adapter.set_failure_status.assert_called()
         logging_mock.assert_called()
+
+
+def test_default_business_exception_handler(job_without_adapter):
+    with patch("pyzeebe.worker.task_router.logger.warning") as logging_mock:
+        with patch("pyzeebe.job.job.Job.set_error_status") as failure_mock:
+            failure_mock.return_value = None
+            error_code = "custom-error-code"
+            exception = BusinessError(error_code)
+            default_exception_handler(exception, job_without_adapter)
+            failure_mock.assert_called_with(mock.ANY, error_code=error_code)
+        logging_mock.assert_called()
```

Comment:
You are checking two things in one test. Maybe split into `test_default_exception_handler_uses_business-exception` and `test_default_exception_handler_warns_of_job_failure`